docs/language/overview.md

```diff
@@ -1572,13 +1548,13 @@ the "in" operator, e.g.,
 
 ##### 7.2.1.6 Predicate Search Term
 
-Any Boolean-valued [function](functions/README.md) like `is()`, `has()`,
-`grep()` etc. and any [comparison expression](#comparisons)
+Any Boolean-valued [function](functions/README.md) like `is`, `has`,
+`grep` etc. and any [comparison expression](#62-comparisons)
 may be used as a search term and mixed into a search expression.
 
 For example,
 ```
-is(<foo>) has(bar) baz x==y+z
+is(<foo>) has(bar) baz x==y+z timestamp > 2018-03-24T17:17:55Z
```

Comment:
I added this comparison with a time literal because @henridf recently happened to ask me how to "filter out events in a given date range" and said he'd not seen how from the docs. I happened to know that the docs already talk a fair amount about literals and comparison functions, but I respect it's not a given that all audiences will put that together, especially if they're newer to the language. I figure it can't hurt to have examples cover common use cases, and this seemed like a harmless place to do that.